src/js/providers/tracks-mixin.ts

```diff
@@ -310,8 +310,13 @@ const Tracks: TracksMixin = {
 
             for (i; i < len; i++) {
                 const track = tracks[i] as TextTrackLike;
-                let trackId: string = track._id || '';
-                if (!trackId) {
+                let trackId: string = track._id || '';                
+
+                if (disableIfOldCaptionsSafari(_tracksById, track, trackId)) {
+                    continue;
+                }
+
+                if (!trackId) {   
```

Comment:
What if we only changed this line to

`if (!trackId && track.inuse !== false) {`

? (and no other changes)